pkg/virt-operator/resource/apply/delete.go

```diff
@@ -634,12 +633,13 @@ func crdHandleDeletion(kvkey string,
 		crdCopy := crd.DeepCopy()
 		controller.AddFinalizer(crdCopy, v1.VirtOperatorComponentFinalizer)
 
-		patchBytes, err := json.Marshal(crdCopy.Finalizers)
+		patchBytes, err := patch.New(
```

Comment:
nit: you can actually remove the `DeepCopy` above and copy only the finalizers now.